sample/build.gradle

```diff
@@ -109,6 +109,12 @@ android {
                 println it
             }
         }
+
+        task('installFromGooglePlay', com.novoda.gradle.command.Install, ['assemble']) {
```

Comment:
Could you remove assemble here to make it consistent with the original install command? Or rename to assembleAndInstallFromGooglePlay